wg-apply/README.md

```diff
@@ -11,6 +11,9 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 Improve the state of declarative object management by fixing `kubectl apply`, moving the logic from the client to the server. This move also enables new/fixed features included in the scope of this working-group: diff, dry-run and prune.
 Resources can be found in [this Google drive folder](https://drive.google.com/drive/folders/1wlpgkS2gFZXdp4x2WlRsfUBxkFlt2Gx0)
 
+## Stakeholder SIGs
+* SIG CLI
+
 ## Meetings
 * Regular WG Meeting: [Tuesdays at 9:30 PT (Pacific Time)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:30&tz=PT%20%28Pacific%20Time%29).
```

Comment:
We've switched to every other tuesday...

Reply:
Fixed in 484c683.